index.html

```diff
@@ -5,242 +5,241 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/4e655bdddb.js" crossorigin="anonymous"></script>
     <link rel="stylesheet" type="text/css" href="./css/styles.css">
     <link rel="stylesheet" type="text/css" href="./css/projects.css">
     <link rel="stylesheet" type="text/css" href="./css/about.css">
     <link rel="stylesheet" type="text/css" href="./css/form.css">
     <title>Personal-Profolio</title>
+    
 </head>
 
 <body>
-
+    <!-- Header Section Starts Here -->
     <header>
-        <nav class="Header">
-
-            <ul class="header-logo">
-                <li class="logo"> My Logo </li>
-                <li class="hamburg-logo"><img src="Img/Hlogo.png" alt="hamburger icon"> </li>
+        <h1 class="logo"><a href="#">My Logo</a></h1>
+        <nav>
+            <svg class="nav-menu-button" width="24" height="24" viewBox="0 0 24 24" fill="none"
+                xmlns="http://www.w3.org/2000/svg">
+                <path fill-rule="evenodd" clip-rule="evenodd"
+                    d="M3 7C3 6.44772 3.44772 6 4 6H20C20.5523 6 21 6.44772 21 7C21 7.55228 20.5523 8 20 8H4C3.44772 8 3 7.55228 3 7ZM3 12C3 11.4477 3.44772 11 4 11L20 11C20.5523 11 21 11.4477 21 12C21 12.5523 20.5523 13 20 13L4 13C3.44772 13 3 12.5523 3 12ZM4 16C3.44772 16 3 16.4477 3 17C3 17.5523 3.44772 18 4 18H20C20.5523 18 21 17.5523 21 17C21 16.4477 20.5523 16 20 16H4Z"
+                    fill="#6070FF" />
+            </svg>
+            <ul class="nav-links">
+                <li class="nav-link">Portfolio</li>
+                <li class="nav-link">About</li>
+                <li class="nav-link">Contact</li>
             </ul>
         </nav>
```

Comment:
- [ ] Great job adding this links in your portfolio, :clap: , you deserve a :clinking_glasses: , howver this navlinks should have an `a` tag so that when a user click on them , they should bring him to the corresponding section :+1: . When you click on `Portfolio` link it should bring him to the project section which represent for you the :love_you_gesture: portfolio section :+1: 

### What to do 
```bash
<li class="nav-link"><a href="id-portfolio-section">Portfolio</a></li>
```

![Screenshot from 2023-01-22 01-30-38](https://user-images.githubusercontent.com/80458199/213894261-2065b0aa-350d-4c52-af64-6e0d0dd11bca.png)